source/mir/random/flex/internal/area.d

```diff
@@ -169,7 +172,8 @@ body
     {
         if (fabs(z) < constants!S.smallExp)
         {
-            area = exp(sh.a) * (iv.rx - iv.lx) * (1 + z / 2 + (z * z) / 6 + (z * z * z) / 24);
+            area = exp(sh.a) * (iv.rx - iv.lx) * (1 + z * S(0.5) + (z * z) * one_div_6
```

Comment:
Can you please explain for me the `exp(sh.a)` part of this formula?

IIUC from Table 2, this is the `f(x0)` component of the area approximation.  I don't follow how it is guaranteed to be `exp(sh.a)` (== exp(alpha) in the paper's notation, right?).